.github/workflows/tests.yaml

```diff
@@ -121,6 +121,8 @@ jobs:
             image: macos-13
           - name: Windows
             image: windows-2022
+          - name: macOS (ARM)
```

Comment:
I am not sure about that. It would add that info to ubuntu and windows jobs as well and it would be pretty useless there since we don't test those systems with multiple architectures

Reply:
ARM Linux runners are coming, so it will very shortly be useful

Reply:
tried that, turns out that `runner` context is not available in `job.name` scope: https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability